new-standalone-service.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+cleanup() {
+    # TODO(mohit): Maybe add a trap for SIGINT.
+    rm -rf template.zip
+}
+
+info() {
+    MESSAGE=$1
+    echo $MESSAGE
+}
+
+debug() {
+    if [[ -z "${DEBUG// }" ]]; then
```

Comment:
Those double slashes ensure that non-empty strings containing only spaces `     ` are also treated as empty.

Without those magical `//`, [[ -z "    " ]] will evaluate to `true`.